### ISSUE TITLE
[Editor] Tick box disappears when multiple objects with different boolean values are selected

### ISSUE THREAD

**Issue by LaraEngebretson** [AtomicGameEngine/AtomicGameEngine#1374]
> Bool variable, when we select a similar object and their value are different (one false and one true) the tick box disappear. (we therefore cannot set all the variable to the same value). Can only multi-assign if they all have the same value. Should be able to do this though.

**Comment by Alan-FGR** [comment]
Yeah, we're gonna need extra art for a 'mixed' checkbox state, right?
